environment/config.go

```diff
@@ -39,18 +38,44 @@ import (
 	"github.com/m3db/m3x/instrument"
 )
 
+const (
+	defaultSDTimeout = 10 * time.Minute
```

Comment:
10 minutes is pretty high. Any reason this can't be order of seconds?

Reply:
+1